packages/core/src/Stores/traders-hub-store.js

```diff
@@ -789,4 +793,12 @@ export default class TradersHubStore extends BaseStore {
         });
         openTopUpModal();
     }
+
+    openRealWalletsUpgrade() {
+        this.is_real_wallets_upgrade_on = true;
+    }
+
+    closeRealWalletsUpgrade() {
```

Comment:
you can merge these two actions :
```suggestion
    toggleRealWalletsUpgrade(value) {
              this.is_real_wallets_upgrade_open = value;
    }
```